### ISSUE TITLE
add Linux support

### ISSUE THREAD

**Issue by petep0p** [MakeraInc/MakeraCAM#22]
please consider adding a Linux version of this software. thank you!

**Comment by manuel-imboden** [comment]
+1 @MakeraInc Are there currently any plans to support Linux in the future?

**Comment by ss32** [comment]
+1 anything but windows

**Comment by bernieke** [comment]
FYI, it seems to install and start fine on wine. Haven't tried a full workflow yet though.

**Comment by markqvist** [comment]
Linux support is what we need! Would really appreciate this!

**Comment by p1um1oco** [comment]
Agreed, linux support would be great, I am trying to eliminate Windows from my life and only have two programs left to go.  Those two run inside a windows VM.

**Comment by jeffc** [comment]
> FYI, it seems to install and start fine on wine. Haven't tried a full workflow yet though.

I got it running under Lutris, which is just Wine plus some free GPU configuration and tweaks, without any issues (or, at least, without any issues that I don't also see on Windows 😢 )

**Comment by markqvist** [comment]
Thanks @jeffc, that's good to know!

Still, a native Linux build would really be preferable. A lot of people in the target demographic of machines like this use Linux for their entire design/production toolchain, and having to switch over to a Windows machine/VM is a major productivity and creativity killer.

**Comment by manuel-imboden** [comment]
@markqvist Hit the nail on the head there – 100% agreed.
I'd love to hear @MakeraInc 's take on this.

**Comment by ss32** [comment]
> I'd love to hear @MakeraInc 's take on this.

Per what I was told by a mod on Discord there is no plan to support Linux.  The app is built on QT so I don't know what's stopping them from supporting Linux other than motivation.

**Comment by jeffc** [comment]
> Still, a native Linux build would really be preferable. A lot of people in the target demographic of machines like this use Linux for their entire design/production toolchain, and having to switch over to a Windows machine/VM is a major productivity and creativity killer.

I totally agree -- I'm one of those people 😆  I have a Windows VM that I use pretty much exclusively for Fusion360. A supported Linux build of MakeraCam would be great, but I will say this as a piece of anecdata: once I set up MakeraCam under Lutris and added a shortcut (in the form of a shell script named `~/bin/makeracam`, I honestly forgot that it *wasn't* a native Linux program. It acts just like one.

> The app is built on QT so I don't know what's stopping them from supporting Linux other than motivation.

As much as I'm saddened by this, I do understand it. As much as it would probably be easy for them to cut a Linux build, *supporting* it would actually be the real trouble.

**Comment by markqvist** [comment]
Even providing a Linux build with the disclaimer that it is "community supported" would be a big plus for a lot of people I think.

If it builds for both Windows and macOS currently, there's not likely to be any real *technical* issues with providing a Linux build, at least.

Even just a raw `x86_64` binary build, and I'm sure community members would be happy to help bundling it to various distribution formats such as AppImage, etc. Put it out, wait about 8.5 seconds, and it's on AUR ;)

**Comment by rvalle** [comment]
No linux support means no real maker product, sorry.

**Comment by petep0p** [comment]
So while it _does_ run under wine, at least on my install (generic wine under Ubuntu 24.04), i consistently get a fatal crash when trying to import an image. So far, everything else seems to be working, though....

**Comment by rvalle** [comment]
mmm Z1 seemed too nice to be true. Will have to wait for Linux support. I would get a herpes in my face if I ever had to use Windows again.

**Comment by evilantishad0w** [comment]
Ditto for linux build; considering it's just Qt compiling for linux should be fairly easy.

**Comment by vahmelk99** [comment]
Any updates?

**Comment by TexZeTech** [comment]
I will also throw my request in for Linux support.

**Comment by petep0p** [comment]
so it's a totally new software title now and loads of work done - but still no Linux version? please guys - i know it's not that far a cry from the Mac version. but please seriously consider the suggestion from earlier in the thread to just release the source for community development if you don't think you have the bandwidth for this...

**Comment by buildacubesat** [comment]
+1 Any word on this would be greatly appreciated.
